nautobot/cloud/tables.py

```diff
@@ -39,6 +40,53 @@ class Meta(BaseTable.Meta):
         )
 
 
+class CloudNetworkTable(BaseTable):
+    pk = ToggleColumn()
+    name = tables.Column(linkify=True)
+    cloud_type = tables.Column(linkify=True)
+    cloud_account = tables.Column(linkify=True)
+    parent = tables.Column(linkify=True)
+    actions = ButtonsColumn(CloudNetwork)
+
+    class Meta(BaseTable.Meta):
+        model = CloudNetwork
+        fields = (
+            "pk",
+            "name",
+            "description",
+            "cloud_type",
+            "cloud_account",
+            "parent",
+            "prefixes",
+            "tags",
+            "actions",
+        )
+        default_columns = (
+            "pk",
+            "name",
+            "description",
+            "cloud_type",
+            "cloud_account",
+            "parent",
+            "actions",
+        )
+
+
+class CloudNetworkPrefixAssignmentTable(BaseTable):
+    cloud_network = tables.Column(
+        verbose_name="Cloud Network",
+        linkify=lambda record: record.cloud_network.get_absolute_url(),
```

Comment:
Is this still needed? Could we just do `linkify=True`?

Reply:
No, this is with `linkify=True`.
![image](https://github.com/nautobot/nautobot/assets/17092348/3712cc50-5e2c-4c08-8fcf-a434589565de)

Reply:
That's weird - looks like it's trying to linkify based on the CloudNetworkPrefixAssignment rather than its `cloud_network` attribute. Maybe because it's the first column in the table?